src/query/expression/src/types.rs

```diff
@@ -607,7 +607,8 @@ pub trait AccessType: Debug + Clone + PartialEq + Sized + 'static {
         std::mem::size_of::<Self::Scalar>()
     }
 
-    fn column_memory_size(col: &Self::Column) -> usize {
+    fn column_memory_size(col: &Self::Column, gc: bool) -> usize {
```

Comment:
Thinking of adding a new function?
https://docs.rs/arrow/latest/arrow/array/struct.ArrayData.html#method.get_buffer_memory_size